src/Symfony/Bridge/Twig/Tests/Node/TransNodeTest.php

```diff
@@ -53,6 +53,6 @@ protected function getVariableGetterWithStrictCheck($name)
             return sprintf('(isset($context["%s"]) || array_key_exists("%s", $context) ? $context["%s"] : (function () { throw new Twig_Error_Runtime(\'Variable "%s" does not exist.\', 0, $this->getSourceContext()); })())', $name, $name, $name, $name);
         }
 
-        return sprintf('($context["%s"] ?? $this->getContext($context, "%s"))', $name, $name, $name);
+        return sprintf('($context["%s"] ?? $this->getContext($context, "%s"))', $name, $name);
```

Comment:
`%1$s`? thus assign `$name` once here as well.

Reply:
Don't you find that notation (`%1$s`) difficult to understand? I always struggle with it.

Reply:
No strong opinion. Just to make things less error prone :) basically the two assignments must always stay the same, thus id prefer a single assignment.

Reply:
@ro0NL I thought about it, but decided not to change. But I agree that `%1$s` is better.

Reply:
I prefer `%1$s`. That's how `sprintf()` should be used to avoid duplications, and we already have some usages in the code base